pyanaconda/modules/localization/installation.py

```diff
@@ -53,16 +55,50 @@ def name(self):
         return "Configure language"
 
     def run(self):
-        self._write_language_configuration(self._lang, self._sysroot)
+        """Run the installation task."""
+        lang = self._get_supported_language()
+        self._write_language_configuration(lang)
+
+    def _get_supported_language(self):
+        """Return a supported locale.
+
+        :return: a locale
+        """
+        if self._is_language_supported(self._lang):
+            return self._lang
+
+        log.debug("The '%s' locale is unsupported.", self._lang)
+        log.debug("Using the '%s' locale as a fallback.", self.LOCALE_FALLBACK)
+        return self.LOCALE_FALLBACK
+
+    def _is_language_supported(self, lang):
+        """Is the language configuration supported?
+
+        The language is considered to be supported if we are not
+        able to determine the supported locales due to missing tools.
+
+        :param lang: a value for the LANG locale variable
+        :return: True if the locale is supported, otherwise False
+        """
+        try:
+            output = execWithCapture("locale", ["-a"], root=self._sysroot)
+        except OSError as e:
+            log.warning("Couldn't get supported locales: %s", e)
+            return True
```

Comment:
I would ask on the related bug if this is good assumption or we should proceed otherwise.

Reply:
I have added a comment to the bug.